### PR TITLE
add doc about openai assistants

### PR DESCRIPTION
**Pull request by qingyun-wu**
## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by qingyun-wu** [comment]
Needs to check the blogpost link once Gagan's blogpost is published.

File: website/docs/Examples/AgentChat.md [hunk 1]
```diff
@@ -39,3 +39,8 @@ Links to notebook examples:
 5. **Agent Teaching and Learning**
    - Teach Agents New Skills & Reuse via Automated Chat - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teaching.ipynb)
    - Teach Agents New Facts, User Preferences and Skills Beyond Coding - [View Notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_teachability.ipynb)
+
+6. **Multi-Agent Chat with OpenAI Assistants in the loop**
```
Comment (sonichi) [inline website/docs/Examples/AgentChat.md:43]: are we missing the most basic one?

File: README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <img src="https://github.com/microsoft/autogen/blob/main/website/static/img/flaml.svg"  width=200>
     <br>
 </p> -->
-:fire: Nov 11: AutoGen experimentally supports OpenAI's Assistants! Checkout the [GPT Assistant Agent](autogen/agentchat/contrib/gpt_assistant_agent.py).
+:fire: Nov 11: AutoGen experimentally supports OpenAI's Assistants! Checkout our [blogpost](https://microsoft.github.io/autogen/blog/2023/11/13/OAI-assistants) for details and examples.
```
Comment (sonichi) [inline README.md:15]:
```suggestion
:fire: Nov 11: OpenAI's Assistants are available in AutoGen and interoperatable with other AutoGen agents! Checkout our [blogpost](https://microsoft.github.io/autogen/blog/2023/11/13/OAI-assistants) for details and examples.
```
Could you double check the term "OpenAI's Assistants"?

Comment (gagb) [reply README.md:15]: OpenAI has simply be using the term "Assistants". So we prepended OpenAI to qualify.

File: website/docs/Examples/AgentChat.md (hunk 1 above)
Comment (gagb) [reply website/docs/Examples/AgentChat.md:43]: https://github.com/microsoft/autogen/blob/main/notebook/agentchat_oai_assistant_twoagents_basic.ipynb

Comment (qingyun-wu) [reply website/docs/Examples/AgentChat.md:43]: Added. Thanks!

**Review by gagb — APPROVED** [review]

**Pull request merged**